extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -1080,7 +1080,7 @@ public String getSupportInfo() {
             // Search for correct panel containing the data
             for (int i = 0; i < panels.size(); i++) {
                 if (panels.getObject(i).getObject("engagementPanelSectionListRenderer")
-                        .getString("panelIdentifier").equals("engagement-panel-macro-markers")) {
+                        .getString("panelIdentifier").equals("engagement-panel-macro-markers-description-chapters")) {
```

Comment:
I've come across two mobile website versions which used the old name while debugging the problem. I suggest to check for both identifiers to have a little safety in case they change it back again.
```diff
--- a/extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java
+++ b/extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java
@@ -1079,8 +1079,10 @@ public class YoutubeStreamExtractor extends StreamExtractor {
 
             // Search for correct panel containing the data
             for (int i = 0; i < panels.size(); i++) {
-                if (panels.getObject(i).getObject("engagementPanelSectionListRenderer")
-                        .getString("panelIdentifier").equals("engagement-panel-macro-markers")) {
+                final String panelIdentifier = panels.getObject(i).getObject("engagementPanelSectionListRenderer")
+                        .getString("panelIdentifier");
+                if (panelIdentifier.equals("engagement-panel-macro-markers")
+                        || panelIdentifier.equals("engagement-panel-macro-markers-description-chapters")) {
                     segmentsArray = panels.getObject(i).getObject("engagementPanelSectionListRenderer")
                             .getObject("content").getObject("macroMarkersListRenderer").getArray("contents");
                     break;

```

Reply:
Yeah the change isn't completed rollout for now. I will apply your suggestion soon.